src/pages/saksbehandling/dokumenter/DokumenterPage.tsx

```diff
@@ -124,29 +113,16 @@ const DokumenterPage = (props: { sak: Sak }) => {
                                 )
                         )
                     )}
-                    <div className={styles.buttonContainer}>
-                        <Button
-                            variant="secondary"
-                            onClick={() => {
-                                history.goBack();
-                            }}
-                        >
-                            <Back />
-                            {formatMessage('knapp.tilbake')}
-                        </Button>
-
-                        <Button
-                            onClick={() => {
-                                handleKontrollsamtaleClick();
-                            }}
-                        >
-                            {formatMessage('knapp.kontrollsamtale.kallInn')}
-                            {RemoteData.isPending(kontrollsamtale) && <Loader />}
-                        </Button>
-                    </div>
-                    {RemoteData.isFailure(kontrollsamtale) && (
-                        <Alert variant="error">{formatMessage('feil.kontrollsamtale.kunneIkkeKalleInn')}</Alert>
-                    )}
+                    <Button
+                        className={styles.tilbakeknapp}
+                        variant="secondary"
+                        onClick={() => {
+                            history.goBack();
+                        }}
```

Comment:
Du som liker mindre kode, här kunde du bara sendt in referansen till funksjonen istedet ;)))

d.v.s `onClick={history.goBack}`

Reply:
Helt enig!!! :D